star-rx72n-firmware/tests/mocks/mock_rx_gpio.c

```
@@ -79,15 +79,15 @@ void mock_gpio_deinit(void)
   memset(&s_mock_gpio, 0, sizeof(s_mock_gpio));
 }
 
-void mock_gpio_set_read_value(gpio_pin_t pin, bool high)
+void mock_gpio_set_read_value(rx_port_pin_t pin, bool high)
```

Clang (14.0.6)

[warning] 2 adjacent parameters of 'mock_gpio_set_read_value' of similar type ('int') are easily swapped by mistake

(bugprone-easily-swappable-parameters)